packages/apps/marmalade-marketplace/src/pages/tokens/[tokenId]/styles.css.ts

```diff
@@ -1,6 +1,6 @@
 import { style } from '@vanilla-extract/css';
 import { tokens } from '@kadena/kode-ui/styles';
-import { deviceColors } from './tokens.css';
+import { deviceColors } from '../../../styles/tokens.css';
```

Comment:
We can get the borderColor from tokens as well, that way they will change with the theming

Reply:
I updated this in another branch.